src/app/server/src/configReader.ts

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs");
+const path = require("path");
+
+class ConfigReader {
+    rootDir: string;
+
+    constructor(rootDir: string) {
+        this.rootDir = rootDir;
+    }
+
+    readConfigFile(...filePath: string[]): Object | null {
```

Comment:
Perhaps you already had this in mind for next iteration but I think we should be using strongly typed configs here ratehr than returning `Object`; that way this parsing process actually validates the config and the user gets a meaningful error message if it doesn't conform to expected structure.